src/main/java/oss/fosslight/service/impl/AutoFillOssInfoServiceImpl.java

```diff
@@ -162,10 +178,6 @@
 				continue;
 			}
 
-			if(downloadLocation.isEmpty()) {
```

Comment:
This part should be added again.
Otherwise, an OSS with a null download location in the OSS List and an item with a null download location in the table matched by Check License.
![image](https://user-images.githubusercontent.com/4284712/145553515-6f08f90b-9c44-4d46-94da-138792926571.png)

Reply:
I agree with your opinion. :smile: